.gitignore

```diff
@@ -72,3 +72,4 @@ fastlane/test_output
 Carthage
 Charts.framework.zip
 ChartsRealm.framework.zip
+.swiftpm
```

Comment:
`.swiftpm` or `.swiftpm/`?

Reply:
Rightfully `.swiftpm/` - let me change that. Did `.swiftpm` cause it does the trick.